app/services/AttachmentService.js

```diff
@@ -225,6 +225,9 @@ export const uploadPunchItemAttachment = (localFileUri,punchId, mimeType, filena
  */
 export const uploadTemporaryAttachment = (localFileUri, type, filename) => {
   let remoteUrl = getUrlForEndpoint("PunchListItem/TempAttachment");
+  if (filename === undefined) {
+    
+  }
```

Comment:
What if filename is undefined? 

Now i am curious! Will there be a sequel ? 

* Cliffhanger codereview? *